libmamba/src/core/activation.cpp

```diff
@@ -1204,17 +1204,17 @@ namespace mamba
 
         for (const std::string& uvar : env_transform.unset_vars)
         {
-            out << "set -e " << uvar << "\n";
+            out << "set -e " << uvar << ";\n";
```

Comment:
Should I remove the newline? It's redundant here!
```suggestion
            out << "set -e " << uvar << ";";
```

Reply:
It probably looks cleaner with a newline, no?